content/chat/rooms/messages.textile

```diff
@@ -46,26 +47,39 @@ The following is the structure of a message:
 
 ```[json]
 {
-  "timeserial": "e91RB0GOQBcvyJ53053074@1718191654749-0",
+  "serial": "e91RB0GOQBcvyJ53053074@1718191654749-0",
   "clientId": "basketLover014",
   "roomId": "basketball-stream",
   "text": "What a shot!",
   "headers": {},
   "metadata": {},
   "createdAt": "2024-06-12T11:37:59.988Z",
+  "latestAction": "message.created",
+  "latestActionSerial": "e91RB0GOQBcvyJ53053074@1718191654749-0",
+  "deletedAt": undefined,
+  "updatedAt": undefined,
+  "latestActionDetails": {},
 }
 ```
 
 The following are the properties of a message:
 
 |_. Property |_. Description |_. Type |
-| timeserial | An Ably-generated ID used to determine the order the message was sent in. | String |
+| serial | An Ably-generated ID used to determine the order the message was sent in. | String |
 | clientId | The client identifier of the user that sent the message. | String |
 | roomId | The name of the room the message was sent to. | String |
 | text | The message contents. | String |
 | headers | Optional headers for adding additional information to a message, such as the relative timestamp of a livestream video, or flagging a message as important. Do not use the headers for authoritative information. There is no server-side validation. When reading headers treat them like user input. | Object |
 | metadata | Optional additional metadata about the message, such as animations, effects or links to other resources such as images. This information is not read by Ably. Do not use metadata for authoritative information. There is no server-side validation. When reading metadata treat it like user input. | Object |
 | createdAt | The time the message was sent. | Date |
+| latestAction | The latest action that was performed on the message, such as a creation, update or deletion. | String |
+| latestActionSerial | The serial of the latest action that was performed on the message. | String |
```

Comment:
Perhaps we should add something here about the fact that serials can be used to determine global ordering?

Reply:
I have added a note below, wdyt? :)